nodebuilder/header/config.go

```diff
@@ -23,13 +25,27 @@ type Config struct {
 	TrustedPeers []string
 
 	Store *store.Parameters
+
+	ServerParameters *p2p_exchange.ServerParameters
+	ClientParameters *p2p_exchange.ClientParameters `toml:",omitempty"`
```

Comment:
Verbosity police. Remove the Parameters Suffix

Reply:
done